src/sqlancer/cockroachdb/CockroachDBOptions.java

```diff
@@ -23,7 +23,7 @@
 import sqlancer.common.oracle.CompositeTestOracle;
 import sqlancer.common.oracle.TestOracle;
 
-@Parameters(separators = "=", commandDescription = "Test CockroachDB")
+@Parameters(separators = "=", commandDescription = "CockroachDB (default port: 26257, default host: localhost)")
```

Comment:
Rather than hard-coding the port and localhost, could you please introduce constants for them that you use in both the description and when setting the values?

For example, in `CockroachDBOptions` you can have a field `final static int DEFAULT_PORT = 26257;` and reference it both here as `"CockroachDB (default port: " + CockroachDBOptions.DEFAULT_PORT + ...` and below in `createDatabase`. By doing so, the help text and actual default values will always be consistent.

Reply:
"CockroachDB (default port: " + CockroachDBOptions.DEFAULT_PORT + ... "   I think it might not work.
It works in `createDatabase`.

Reply:
Indeed, the current help text and default value can be consistent here. 
Thank you for your suggestions.  
TBR @mrigger

Reply:
Sorry again for the late response - I had some full-day obligations in the last couple of days, and will be able to respond much faster in the future.

Why does using the constant in the annotation text not work?